wherehows-web/app/routes/application.js

```diff
@@ -31,6 +32,19 @@ export default Route.extend(ApplicationRouteMixin, {
   },
 
   /**
+   * __It reloads the Ember.js application__ by redirecting the browser
+   * to the application's root URL so that all in-memory data (such as Ember
+   * Data stores etc.) gets cleared.
+   * @override ApplicationRouteMixin.sessionInvalidated
+   * @link https://github.com/simplabs/ember-simple-auth/issues/1048
+   */
+  sessionInvalidated() {
+    if (!testing) {
+      window.location.replace('/');
```

Comment:
I know this is a cost, but it's apt for this use case. Another option, would be to tag each in-memory of cached data item that was user sensitive, and manually, invalidate each one when a session gets invalidated. or persist each shared data item in persistent storage, like localStorage etcetera, but that is probably such a hassle it wouldn't be worth it for this use case